vibe_code_apps_22/scripts/eplus_heating_dsm_farm.py

```
@@ -246,14 +246,68 @@ def _eligible_idf(root: Path) -> Path:
+def build_scenarios(
+    *,
+    smoke: bool = False,
+    medium: bool = False,
+    crossed: bool = False,
+    n_weather_days: int = 40,
+) -> list[dict]:
+    """Paired scenarios: one baseline run per day + DSM arms.
+
+    Modes
+    -----
+    - smoke: ~6 days × 1 DSM strategy (screening; underpowered)
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)